index.html

```diff
@@ -213,45 +200,52 @@ <h4>
             targets are available, or the identity of the chosen target.
           </p>
           <div class="note">
-            <a for="Navigator"><code>share</code></a> always shows some form of
-            UI, to give the user a choice of application and get their approval
-            to invoke and send data to a potentially native application (which
-            carries a security risk). For this reason, user agents are
-            prohibited from showing any kind of "always use this target in the
-            future" option, or bypassing the UI if there is only a single share
-            target.
+            <a>share()</a> always shows some form of UI, to give the user a
+            choice of application and get their approval to invoke and send
+            data to a potentially native application (which carries a security
+            risk). For this reason, user agents are prohibited from showing any
+            kind of "always use this target in the future" option, or bypassing
+            the UI if there is only a single share target.
           </div>
         </section>
       </section>
-      <section data-dfn-for="ShareData">
+      <section data-dfn-for="ShareData" data-link-for="ShareData">
         <h3>
           <code>ShareData</code> dictionary
         </h3>
         <pre class="idl">
           dictionary ShareData {
-            USVString title;
             USVString text;
+            USVString title;
```

Comment:
Hmm, why did the order change? Are you just making it alphabetical? I'd prefer to keep it in order title > text > url, since that's the order I think developers should supply them if supplying them all. (That's the order that a receiver will normally display them.)

(Note that WebIDL dictionaries don't care what the order is, so this decision isn't normative, just presentational.)

Reply:
Ping?

Reply:
ah, sorry... just saw this... was alphabetical, will revert.